src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -709,6 +707,14 @@ public class SimpleEvents {
 					.requiredPlugins("Paper 1.16+");
 		}
 
+		Skript.registerEvent("Server Broadcast", SimpleEvent.class, BroadcastMessageEvent.class, "[server] broadcast")
```

Comment:
Does this event work for skript's broadcast effect?

Reply:
No